src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int taskNumber = 6;
+    private static final int halfOfTaskNumber = taskNumber / 2 - 1;
```

Comment:
it is better to change style of naming
Please, look at [Constant names](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)